packages/utils/src/browser/isWebGLSupported.ts

```diff
@@ -30,9 +30,9 @@ export function isWebGLSupported(): boolean
                 let gl = (
                     canvas.getContext('webgl', contextOptions)
                     || canvas.getContext('experimental-webgl', contextOptions)
-                ) as WebGLRenderingContext;
+                ) as WebGLRenderingContext | null;
 
-                const success = !!(gl && gl.getContextAttributes().stencil);
+                const success = !!(gl && gl.getContextAttributes()?.stencil);
```

Comment:
```suggestion
                const success = !!gl?.gl.getContextAttributes()?.stencil;
```

Reply:
Is this supposed to be `!!gl?.getContextAttributes()?.stencil;`

I see there is an extra `gl in `gl?.gl...`